data/gegno/ships.txt

```diff
@@ -0,0 +1,954 @@
+
+ship Conglomerate
+	sprite ship/conglomerate
+	thumbnail thumbnail/conglomerate
+	swizzle 0
+	attributes
+		category "Heavy Freighter"
+		licenses
+			"Gegno Civilian"
+		cost 1692000
+		mass 760
+		bunks 166
+		"cargo space" 400
+		drag 8.9
+		"engine capacity" 136
+		"fuel capacity" 400
+		"heat dissipation" 0.7
+		hull 13300
+		"outfit space" 240
+		"required crew" 68
+		shields 0
+		"weapon capacity" 18
+		weapon
+			"blast radius" 124
+			"shield damage" 960
+			"hull damage" 1160
+			"hit force" 1650
+	outfits
+		"C7 Brawl Core"
+		Hyperdrive
+		"R02 Battlezone Battery"
+		"RG18 Torch Steering"
+		"RG18 Torch Thruster"
+	engine -32.5 215
+	engine 32.5 215
+	leak leak 30 40
+	leak flame 80 20
+	explode "huge explosion" 10
+	explode "large explosion" 30
+	explode "medium explosion" 38
+	explode "small explosion" 26
+	"final explode" "final explosion large"
+	description "Over time the Gegno needed larger vessels to carry materials from new worlds alike- ones that would be more efficient and more efficient to build than older outdated City-Ships. Conglomerates are built to haul, moving large amounts of materials and personnel across Gegno space."
+
+ship Gypsum
+	sprite ship/gypsum
+	thumbnail thumbnail/gypsum
+	swizzle 0
+	attributes
+		category "Light Freighter"
+		licenses
+			"Gegno Civilian"
+		cost 1692000
+		mass 380
+		bunks 40
+		"cargo space" 104
+		drag 3
+		"engine capacity" 120
+		"fuel capacity" 500
+		"heat dissipation" 0.8
+		hull 5200
+		"outfit space" 240
+		"required crew" 18
+		shields 0
+		"weapon capacity" 42
+		weapon
+			"blast radius" 72
+			"shield damage" 320
+			"hull damage" 620
+			"hit force" 720
+	outfits
+		"C7 Brawl Core"
+		Excavator
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"RG15 Torch Steering"
+		"RG18 Torch Steering"
+		"RG18 Torch Thruster"
+	engine 0 124
+	gun 0 -117 Excavator
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 14
+	explode "medium explosion" 24
+	explode "small explosion" 26
+	explode "tiny explosion" 10
+	"final explode" "final explosion medium"
+	description "To fuel their exponentionally increasing construction rate and need for materials, many Miner-class ships were devised for space usage. Rather than re-use older ship designs, Miners would be designed to focus only on the purpose of mining. The Gypsum platform is not much more than a flying drill with cargo containers, engines, and a bridge attached behind it, making it cost-efficient and easily manufacturable."
+	description `Compared to other vessels in the galaxy, the Gypsum is much slower due to the Gegno's technology level, and has a harder time keeping up with faster asteroids. For those asteroids it can catch up to, the powerful excavator drill is able to grind materials out with ease.`
```

Comment:
```suggestion
	description `	Compared to other vessels in the galaxy, the Gypsum is much slower due to the Gegno's technology level, and has a harder time keeping up with faster asteroids. For those asteroids it can catch up to, the powerful excavator drill is able to grind materials out with ease.`
```